rust-src/concordium_base/src/ecvrf/mod.rs

```diff
@@ -262,7 +262,12 @@ mod tests {
 
             // Test serialization of generated secret scalar
             let x = expanded_sk.key;
-            assert_eq!(x, Scalar::from_bits(x_bytes));
+            // TODO: we use deprecated `from_bits` here, this also requires enabling the
+            // `legacy_compatibility` feature for the `ed25519-dalek` dependency. Maybe ther
+            // is a different way of implementing this.
+            #[allow(deprecated)]
+            let scalar = Scalar::from_bits(x_bytes);
+            assert_eq!(x, scalar);
```

Comment:
So this is just in a test, so we can only enable this legacy feature as part of a `dev-dependency`.

But instead, could we not just check `x.to_bytes() == x_bytes`?

Reply:
Changed to use `from_bytes_mod_order()`